README.md

```diff
@@ -66,3 +66,23 @@ and follow the above installation instructions as usual.
 
 ### Firmware Demo
 [![asciicast](https://asciinema.org/a/96442.png)](https://asciinema.org/a/96442)
+
+## OpenSSL (Linux)
+In some cases users may want to use the switchtec-cli with a different/custom 
+version of OpenSSL that is not the default version packaged with the OS, whose 
+library files exist elsewhere in the host system and therefore is not expected 
+by the current switchtec-cli build system.
+
+Users have observed the below error (or similar shared library errors) when 
+using OpenSSL commands after compiling the codebase.
```

Comment:
I'm a bit confused by this paragraph. Under what circumstances is this error being emitted? Is compiling a custom version of libssl the solution to whatever caused this? 

Does switchtec-user depend on a specific version of libssl and commonly used systems have a version that is too old? What versions of libssl are affected? What operating systems are affected?